### PR TITLE
Frontend

### PR DESCRIPTION
**Pull request by gobyrne**

### CODE REVIEW

File: app/bower_components/angularjs/.bower.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
```
Comment (kheaactua) [inline app/bower_components/angularjs/.bower.json]: Please remove this file (everything in `bower_components/`)

File: app/index.html [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <!doctype html>
 <html ng-app="diplomacy">
   <head>
-    <script type="text/javascript" src="https://ajax.googleapis.com/ajax/libs/angularjs/1.3.15/angular.min.js"></script>
-	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
-	<link rel="stylesheet" type="text/css" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" />
+    <script type="text/javascript" src="/app/bower_components/jquery/dist/jquery.js"></script>
+    <script type="text/javascript" src="/app/bower_components/angularjs\angular.js"></script>
+	<link rel="stylesheet" type="text/css" href="/app/bower_components/bootstrap/dist/css/bootstrap.css" />
```
Comment (kheaactua) [inline app/index.html]: Might want to use the `.min.js` versions.  Well, up to you.  These are easier to debug, but for production we'd want to change.  (Well, if we were in a proper production mode, we'd have a system that built them automatically, and wouldn't have to think about it.)

File: app/bower_components/jquery/src/wrap.js [hunk 3]
```diff
@@ -0,0 +1,79 @@
+define([
+	"./core",
+	"./core/init",
+	"./manipulation", // clone
+	"./traversing" // parent, contents
+], function( jQuery ) {
+
+jQuery.fn.extend({
```
Comment (kheaactua) [inline app/bower_components/jquery/src/wrap.js]: Can you comment what this does/what it's for?

File: app/index.html (hunk 2 above)
Comment (gobyrne) [inline app/index.html]: Good point, but lets leave it for now... it might actually help for something.

File: app/index.html [hunk 4]
```diff
@@ -1,97 +1,102 @@
...
-					</blockquote>
-				</form>
-			</li>
-		</ul>
-		<input class="btn btn-primary" type="submit" ng-click="sendOrders(matchId, empireId, 'move 2 1')">
+					</form>
+				</li>
+			</ul>
+		</div>
+		<input class="btn btn-primary" type="submit" onclick="sendForms()">
```
Comment (kheaactua) [inline app/index.html]: This is outside of your controller/scope

File: app/script.js [hunk 5]
```diff
@@ -0,0 +1,16 @@
+var server = 'http://diplomacy3.asilika.com';
+
+sendForms = function(){
```
Comment (kheaactua) [inline app/script.js:2]: You gotta document what these do better

**Pull request merged**